stripe/src/main/java/com/stripe/android/StripePaymentController.kt

```diff
@@ -439,14 +439,14 @@ internal class StripePaymentController internal constructor(
                         )
                     )
 
-                    val redirectData = stripeIntent.redirectData
+                    val redirectData = stripeIntent.nextActionData as StripeIntent.NextActionData.RedirectToUrl
```

Comment:
can we use smart casting via local variable?

Reply:
done